Update configuration settings table to include the stack join configuration (#5478)
OpenLiberty/docs: modules/ROOT/pages/log-trace-configuration.adoc

|
|This setting specifies the  maximum size (in MB) that a log file can reach before it is rolled. Setting the value to `0` disables log rolling. The default value is `20`. The `console.log` does not roll so this setting does not apply.

|stackJoin
|com.ibm.ws.logging.stackJoin
|WLP_LOGGING_STACK_JOIN

> |This attribute, when set to `true` will merge stack traces emitted to standard-out/standard-error into a single log event.  By default this value is set to `false` and each individual line of the stack trace are emitted as individual log events.

Reviewer: ```suggestion
| When this attribute is set to `true`,  stack traces that are emitted to standard-out and standard-error are merged into a single log event.  By default, this value is set to `false` and each individual line of the stack trace is emitted as an individual log event.
```
